File: split-bill-api/src/main/java/com/shashankbhat/splitbill/dto/groups/GroupsAllDataDto.java
package com.shashankbhat.splitbill.dto.groups;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GroupsAllDataDto {
    List<GroupsEntityDto> data;
}

File: app/src/main/java/com/shashankbhat/splitbill/database/remote/entity/GroupsAllDataDto.kt
package com.shashankbhat.splitbill.database.remote.entity

import com.shashankbhat.splitbill.database.local.dto.group_list.GroupListDto
import kotlinx.serialization.Serializable

@Serializable
class GroupsAllDataDto {
    var data: List<GroupListDto>? = null
}
